cake-autorate.sh

```diff
@@ -824,7 +825,7 @@ maintain_pingers()
 	# this initiates the pingers and monitors reflector health, rotating reflectors as necessary
 
  	trap '' INT
-	trap 'terminate_reflector_maintenance=1' TERM EXIT
+	trap 'kill_maintain_pingers' TERM EXIT
```

Comment:
The reason for indirect termination is because replace_reflector kills and restarts fping / ping. So if this strikes right before the pid is updated then we can be left with residual unkilled fping / ping processes.

Reply:
Yes, makes sense. I'll see what to do then..